src/__test__/productList.test.js

```diff
@@ -0,0 +1,18 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import { Provider } from 'react-redux';
+import App from '../App';
+import store from '../redux/store';
+
+describe('test PlayersList component', () => {
+  test('check whether the component renders correctly', () => {
+    const tree = render(
+      <React.StrictMode>
+        <Provider store={store}>
+          <App />
+        </Provider>
+      </React.StrictMode>,
+    );
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
You have done a great job with this unit test but you should try adding some more unit tests that check for several components of the app and some pure functions in Redux code 🚀 Kindly add these tests to your code so that your project meets the requirements 👍

Reply:
- [ ] Like the previous reviewer suggested, please try to write some more tests to your application 👍